docs/2.0.0-Beta/litmus-2.0.0-Beta.yaml

```diff
@@ -143,6 +143,9 @@ rules:
   - apiGroups: [litmuschaos.io]
     resources: [chaosengines, chaosexperiments, chaosresults]
     verbs: [get, create, update, patch, delete, list, watch, deletecollection]
+  - apiGroups: [apiextensions.k8s.io]
+    resources: [customresourcedefinitions]
+    verbs: [list, get]
```

Comment:
can we remove it from ns where you have added?

Reply:
Yes, it was going to be added in this PR only. I have reverted my changes.. It is added only for cluster scope now.